spec/lib/billy/cache_spec.rb

```diff
@@ -78,7 +78,7 @@
       end
 
       it 'More specifically, the cache keys should be identical for the 2 analytics urls' do
-        identical_cache_key = 'post_5fcb7a450e4cd54dcffcb526212757ee0ca9dc17'
+        identical_cache_key = 'post_www.example-analytics.com_5fcb7a450e4cd54dcffcb526212757ee0ca9dc17'
```

Comment:
Actually, I think we want to keep host out of here by default.  See this user's usage: https://github.com/oesmith/puffing-billy/pull/107#issuecomment-101219797